activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+*   `travel/travel_to` travel time helpers, now raise on nested calls, 
+     as this can lead to confusing time stubbing.
+       
+     Instead of:
+     
+         travel_to 2.days.from_now do
+           # 2 days from today
+           travel_to 3.days.from_now do
+             # 5 days from today
+           end          
+         end
+
+     preferred way to achieve above is:
+
+         travel_to 2.days.from_now
+         # 2 days from today
+         
+         travel_back
+         travel_to 5.days.from_now 
```

Comment:
Won't `travel_back` be what users want to call 90% of the times if they add a second `travel_to` in tests? Then shouldn't we just call it automatically for them? We could add an `reset: false` opt-out to `travel` and `travel_to`?